server/netty-server/src/main/scala/sttp/tapir/server/netty/package.scala

```diff
@@ -25,4 +37,53 @@ package object netty {
       run(routes.toList)
     }
   }
+
+  implicit class RichOptionalNettyResponse(val r: Option[NettyResponse]) {
```

Comment:
I think since this is used only once maybe it would be enough to make this a private method in the interpreter?

Reply:
sorry, server handler

Reply:
Moved all `Rich*` implicit classes to the handler since they are used only there + made them private.